build.gradle

```diff
@@ -361,6 +360,8 @@ dependencies {
   implementation group: 'com.azure', name: 'azure-core', version: versions.azureCore
   implementation group: 'com.azure', name: 'azure-messaging-servicebus', version: versions.azureMessagingServicebus
 
+  implementation group: 'org.apache.qpid', name: 'qpid-jms-client', version: versions.qpidJmsClient
```

Comment:
(Minor) makes more sense grouped with the other apache dependencies

Reply:
Yeah good shout